### DISCUSSION TITLE
template message

### DISCUSSION THREAD

**Discussion by ridone713** [filamentphp/filament#12332] [Help]
### Package

Actions

### Package Version

v3.0.0

### How can we help you?

how do I add a template message to that WhatsApp number?

 Tables\Actions\Action::make('wa')
                    ->label(false)
                    ->color('success')
                    ->button()->link()->icon('heroicon-o-chat-bubble-left-ellipsis')
                    ->size(ActionSize::Medium)
                    ->url(fn (Tagihan $record): string => "https://wa.me/{$record->penghunis->no_hp}")
                    ->openUrlInNewTab(),

like this

![wa2](https://github.com/filamentphp/filament/assets/164133045/cf6f289b-8bd8-4d06-927e-3710a3ed43e6)

**Comment by leandrocfe** [comment]
`?text=your_message`

->url(fn (Tagihan $record): string => "[https://wa.me/{$record->penghunis->no_hp}](https://wa.me/%7B$record-%3Epenghunis-%3Eno_hp%7D)?text=your_message”)
